mkdocs/themes/readthedocs/base.html

```diff
@@ -122,34 +123,36 @@
 
     <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">
 
-      {# MOBILE NAV, TRIGGLES SIDE NAV ON TOGGLE #}
-      <nav class="wy-nav-top" role="navigation" aria-label="top navigation">
-        <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
-        <a href="{{ nav.homepage.url|url }}">{{ config.site_name }}</a>
+      {#- MOBILE NAV, TRIGGLES SIDE NAV ON TOGGLE #}
+      <nav class="wy-nav-top" role="navigation" aria-label="{% trans %}Top Navigation{% endtrans %}" {% if config.theme.style_nav_header_background %} style="background: {{config.theme.style_nav_header_background}}" {% endif %}>
+        {%- block mobile_nav %}
+          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
+          <a href="{{ nav.homepage.url|url }}">{{ config.site_name }}</a>
+        {% endblock %}
       </nav>
 
-      {# PAGE CONTENT #}
+      {#- PAGE CONTENT #}
       <div class="wy-nav-content">
         <div class="rst-content">
-          {% include "breadcrumbs.html" %}
-          <div role="main">
-            <div class="section">
+          {%- include "breadcrumbs.html" %}
+          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
```

Comment:
I added the extra metadata because it was missing from the MkDocs fork. I'm not sure the benefit, but it's probably something that is for accessibility? It wasn't any structural changes, just the following added.
`class="document" itemscope="itemscope" itemtype="http://schema.org/Article"`
` itemprop="articleBody"`
https://github.com/Sparticuz/mkdocs/blob/master/mkdocs/themes/readthedocs/base.html#L151-L152

Reply:
A repo search for `itemprop` didn't reveal anything, so unless anyone objects, I always prefer complete syntax.